server/src/main/java/org/opensearch/index/query/FuzzyQueryBuilder.java

```diff
@@ -181,6 +186,9 @@ public FuzzyQueryBuilder(StreamInput in) throws IOException {
         maxExpansions = in.readVInt();
         transpositions = in.readBoolean();
         rewrite = in.readOptionalString();
+        if (in.getVersion().after(Version.V_2_16_0)) {
```

Comment:
We really need to pinpoint 2.17.0 (current `2.x`) as a baseline (in a course of events when 2.16.1 could be released at any time, this check will break since we won't be backporting this change to 2.16.x line):

```suggestion
        if (in.getVersion().onOrAfter(Version.V_2_17_0)) {
```